nanoFirmwareFlasher.Library/OutputWriter.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading;
+
+namespace nanoFramework.Tools.FirmwareFlasher
+{
+    /// <summary>
+    /// Writes output to the console. In unit tests the output can be directed to
```

Comment:
As you know, these comments are for Intelisense and documentation. They should stick to a clear and concise description of the class/method/etc. There is no need to add information about the motivation or judgment on code and unrelated matters. Those belong to the PRs holding the changes.

Reply:
Ok, changed the comment.

Reply:
Now it looks OK!